veil-macros/src/lib.rs

```diff
@@ -1,87 +1,44 @@
+//! Macros for `veil`
+
+#![cfg_attr(not(docsci), warn(missing_docs))]
+#![cfg_attr(not(docsci), warn(rustdoc::broken_intra_doc_links))]
+#![cfg_attr(docsci, deny(missing_docs))]
+#![cfg_attr(docsci, deny(rustdoc::broken_intra_doc_links))]
```

Comment:
```suggestion
#![warn(missing_docs)]
```

Same thing as before, I forgot to leave a comment here